webapp/src/main/java/org/opentestsystem/rdw/admin/service/GroupService.java

```diff
@@ -8,9 +8,27 @@
 
 import java.util.Set;
 
-@PreAuthorize("hasAuthority('PERM_GROUP_READ')")
+@PreAuthorize("hasAuthority('PERM_GROUP_WRITE')")
```

Comment:
Aside from service lock-down with these annotations I think we can more restrictively update the spring *SecurityConfig.java file to only permit access to "/*" to users with permission "PERM_GROUP_WRITE"